suitcase/pizza-box/handlers.py

```diff
@@ -0,0 +1,116 @@
+import uuid
+import humanize
+import pandas as pd
+import os
+from databroker.tests.utils import temp_config
+from databroker import Broker
+from databroker.assets.handlers_base import HandlerBase
+
+# this will create a temporary databroker object with nothing in it
+db = Broker.from_config(temp_config())
+
+fc = 7.62939453125e-05
```

Comment:
What's this magic number? Can some related documentation be provided?

Reply:
These are values that are needed to convert from the hardware values to the measured data in Volts. As far as I understand, every pizza box has the same transformation (we also tested this with the new pizza boxes at QAS).

We can say:

> These transformations are the transformations necessary to convert the hex values that come from the ADC's into units of Volts.

@lcdesilva can you comment with more details on this? Can you supply the pizza box model number or something we can write to identify the pizza box? In the case a newer pizza box comes out that actually has different values.